build.cake

```diff
@@ -725,22 +716,47 @@ void RunTest(FilePath exePath, DirectoryPath workingDir, string testAssembly, st
         errorDetail.Add(string.Format("{0} returned rc = {1}", exePath, rc));
 }
 
-void RunDotnetCoreTests(FilePath exePath, DirectoryPath workingDir, string framework, ref List<string> errorDetail)
+void RunDotnetCoreNUnitLiteTests(FilePath exePath, DirectoryPath workingDir, string framework, ref List<string> errorDetail)
 {
     RunDotnetCoreTests(exePath, workingDir, arguments: null, framework, ref errorDetail);
 }
 
 void RunDotnetCoreTests(FilePath exePath, DirectoryPath workingDir, string arguments, string framework, ref List<string> errorDetail)
 {
+    //Filename is first arg if running on NUnit Console of exePath if running NUnitLite tests
```

Comment:
I'm not sure I understand this. Should `Console of exePath` be `Console or exePath` ?

Reply:
Sorry, it should be "or"! This is a bit of a hack anyway...maybe that's a sign that I should refactor it better. 🙂